xhr.bs

```diff
@@ -1282,19 +1283,32 @@ transfer-encoding: chunked</code></pre>
  <li><p>Return <var>mimeType</var>.
 </ol>
 
-<p>The <dfn id=override-mime-type>override MIME type</dfn> is initially null and can get a value if
-{{overrideMimeType()}} is invoked. <dfn id=final-mime-type>Final MIME type</dfn> is the
+<p>The <dfn id=override-mime-type>override MIME type</dfn> is initially null and can get a value
+when {{overrideMimeType()}} is invoked. <dfn id=final-mime-type>Final MIME type</dfn> is the
```

Comment:
Nit: You used "the" for override MIME type but left it out for "final MIME type" and "final charset". (I prefer including it.)